.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "FSharp.fsacRuntime": "net"
-}
+  "editor.formatOnSave": true,
+  "ocaml.packageManager": "esy"
```

Comment:
is this necessary?

Reply:
I committed `editor.formatOnSave` on accident. I don't know about `ocaml.packageManager`. I'd just ignore `.vscode/settings`, what do you think?

Reply:
Removing the settings.json is fine. I noticed it was already checked in and thought it should updated with relevant ones.